app/views/courses/index.html.erb

```diff
@@ -26,9 +30,13 @@
             <% end %>
 
             <% if category != :completed %>
-              <div class="collection red darken-3">
+              <div class="collection red darken-3 date">
                 <% course.current_assessments.ordered.each do |asmt| %>
-                  <%= link_to asmt.display_name, course_assessment_path(course, asmt), class: "collection-item grey-text text-darken-4" %>
+                  <%= link_to course_assessment_path(course, asmt), class: "collection-item grey-text text-darken-4 date" do %>
+                  <%= asmt.display_name %>
+                  <p class="date"><b>Start:</b>&ensp;<%= asmt.date_to_s(asmt.start_at) %>&ensp;|
```

Comment:
seems like this code is mostly reused, any way to make this into like a partial?